src/signalingclient/SignalingClientJoin.ts

```diff
@@ -9,15 +9,11 @@ import ApplicationMetadata from '../applicationmetadata/ApplicationMetadata';
 export default class SignalingClientJoin {
   /** Initializes a SignalingClientJoin with the given properties.
    *
-   * // maxVideos is deprecated as the video sending/receiving capacity limits
-   * // for a meeting will be controlled by the backend.
-   * @param maxVideos Usage of maxVideos variable will result into no-op.
    * @param sendBitrates Whether the server should send Bitrates messages.
    * @param applicationMetadata [[ApplicationMetadata]].
    */
 
   constructor(
-    public maxVideos: number,
     public sendBitrates: boolean,
```

Comment:
This should always be true.

Reply:
Update to remove sendBitrates and send server side bwe.